src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -12,7 +12,15 @@
                 i18n>Grafana Dashboard doesn't exist. Please refer to
   <cd-doc section="grafana"></cd-doc> on how to add dashboards to Grafana.</cd-alert-panel>
 
-<ng-container *ngIf="grafanaExist && dashboardExist">
+  <cd-alert-panel type="info"
+                  *ngIf="!secure"
+                  i18n>The embedded Grafana Dashboard was blocked due to a certificate error, please click <a href="https://172.20.0.4:3000/"
```

Comment:
@nSedrickm This is a Nice improvement, Thanks!
But why grafana URL is hardcoded here, It should fetch from the cluster itself, right?

Reply:
@sunilangadi2  grafana URL is now dynamic

Reply:
@nSedrickm Looks good, But some of the ceph dashboard tests got failed can you please check that?

Reply:
Hi @sunilangadi2 I will